src/components/NamePicker.tsx

```diff
@@ -1,10 +1,76 @@
-import React from 'react';
-import TextField from '@material-ui/core/TextField';
+import React, { useState, MouseEvent, ChangeEvent } from 'react';
 import Grid from '@material-ui/core/Grid';
+import Popover from '@material-ui/core/Popover';
+import TextField from '@material-ui/core/TextField';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
 import { makeStyles } from '@material-ui/core/styles';
 
 export default function NamePicker(): JSX.Element {
   const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
```

Comment:
This can be a contentious issue, but the [Typescript coding guidelines](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined) say "Use `undefined`. Do not use `null`." I try to follow that, but occasionally need to use `null`.

I find it is also more convenient in React to use `undefined` instead of `null`, though often they can be used interchangeably. For example, with `undefined`, this statement becomes simply:
`const [anchorEl, setAnchorEl] = useState<HTMLElement>();`
The compiler figures out that `anchorEl` can be either an `HTMLElement` or `undefined` and will do `undefined` checking for you.

I'll make comments below about what would change if you used `undefined` instead of `null`.

Reply:
Can be changed to undefined with no problems and will take note of it in the future, thank you!